alt-fuel-finder/app/controllers/search_controller.rb

```diff
@@ -2,7 +2,6 @@ class SearchController < ApplicationController
 
 
   def index
-    conn = NrelService.new
-    @stations = conn.nearest_stations_by_zip(params["q"])[0..9]
```

Comment:
I moved the results limiting function to an optional argument in my NrelService Faraday call